static/js/components/subject_page/category.tsx

```diff
@@ -88,34 +90,51 @@ function renderBlocks(
       case "DISASTER_EVENT":
         return (
           <ErrorBoundary key={id}>
-            <DisasterEventBlock
+            <BlockContainer
               id={id}
-              place={props.place}
-              enclosedPlaceType={props.enclosedPlaceType}
               title={block.title}
               description={block.description}
               footnote={block.footnote}
-              columns={block.columns}
-              eventTypeSpec={props.eventTypeSpec}
-              showData={props.showData}
-            />
+              place={props.place}
+            >
+              <DisasterEventBlock
+                id={id}
+                place={props.place}
+                enclosedPlaceType={props.enclosedPlaceType}
+                title={block.title}
+                description={block.description}
+                footnote={block.footnote}
+                columns={block.columns}
+                eventTypeSpec={props.eventTypeSpec}
+                showData={props.showData}
+              />
+            </BlockContainer>
           </ErrorBoundary>
         );
       default:
         return (
           <ErrorBoundary key={id}>
```

Comment:
Maybe have a bug to add <ErrorBounday> to all the components

Reply:
do you mean create an issue? & add what to all the components?

Reply:
Yep, as an issue. I sometimes see error message to ask for ErrorBoundary, it would be nice to have it for each of the tile?

Reply:
I see, ok, will create an issue!